plugins/org.eclipse.emf.mwe.utils/src/org/eclipse/emf/mwe/utils/Writer.java

```diff
@@ -63,6 +71,18 @@ public void setCloneSlotContents(final boolean b) {
 	public void setOPTION_SCHEMA_LOCATION_IMPLEMENTATION(final boolean option_schema_location_implementation) {
 		OPTION_SCHEMA_LOCATION_IMPLEMENTATION = option_schema_location_implementation;
 	}
+	
+	@Override
+	public void checkConfiguration(Issues issues) {
+		super.checkConfiguration(issues);
+		if (encoding != null) {
+			try {
```

Comment:
java.nio.charset.Charset.isSupported(String) ?

Reply:
oh yes, even better. Was not aware of that, thanks!

Reply:
updated